internal/bminventory/inventory_test.go

```diff
@@ -13409,31 +13422,31 @@ func validateNetworkConfiguration(cluster *models.Cluster, clusterNetworks *[]*m
 	if clusterNetworks != nil {
 		ExpectWithOffset(1, cluster.ClusterNetworks).To(HaveLen(len(*clusterNetworks)))
 		for index := range *clusterNetworks {
-			Expect(cluster.ClusterNetworks[index].ClusterID).To(Equal(*cluster.ID))
-			Expect(cluster.ClusterNetworks[index].Cidr).To(Equal((*clusterNetworks)[index].Cidr))
-			Expect(cluster.ClusterNetworks[index].HostPrefix).To(Equal((*clusterNetworks)[index].HostPrefix))
+			ExpectWithOffset(1, cluster.ClusterNetworks[index].ClusterID).To(Equal(*cluster.ID))
+			ExpectWithOffset(1, cluster.ClusterNetworks[index].Cidr).To(Equal((*clusterNetworks)[index].Cidr))
+			ExpectWithOffset(1, cluster.ClusterNetworks[index].HostPrefix).To(Equal((*clusterNetworks)[index].HostPrefix))
 		}
 		// TODO(MGMT-9751-remove-single-network)
-		Expect(cluster.ClusterNetworkCidr).To(Equal(""))
-		Expect(cluster.ClusterNetworkHostPrefix).To(Equal(int64(0)))
+		ExpectWithOffset(1, cluster.ClusterNetworkCidr).To(Equal(""))
+		ExpectWithOffset(1, cluster.ClusterNetworkHostPrefix).To(Equal(int64(0)))
 	}
 	if serviceNetworks != nil {
-		Expect(cluster.ServiceNetworks).To(HaveLen(len(*serviceNetworks)))
+		ExpectWithOffset(1, cluster.ServiceNetworks).To(HaveLen(len(*serviceNetworks)))
 		for index := range *serviceNetworks {
-			Expect(cluster.ServiceNetworks[index].ClusterID).To(Equal(*cluster.ID))
-			Expect(cluster.ServiceNetworks[index].Cidr).To(Equal((*serviceNetworks)[index].Cidr))
+			ExpectWithOffset(1, cluster.ServiceNetworks[index].ClusterID).To(Equal(*cluster.ID))
+			ExpectWithOffset(1, cluster.ServiceNetworks[index].Cidr).To(Equal((*serviceNetworks)[index].Cidr))
 		}
 		// TODO(MGMT-9751-remove-single-network)
-		Expect(cluster.ServiceNetworkCidr).To(Equal(""))
+		ExpectWithOffset(1, cluster.ServiceNetworkCidr).To(Equal(""))
 	}
 	if machineNetworks != nil {
-		Expect(cluster.MachineNetworks).To(HaveLen(len(*machineNetworks)))
+		ExpectWithOffset(1, cluster.MachineNetworks).To(HaveLen(len(*machineNetworks)))
 		for index := range *machineNetworks {
-			Expect(cluster.MachineNetworks[index].ClusterID).To(Equal(*cluster.ID))
-			Expect(cluster.MachineNetworks[index].Cidr).To(Equal((*machineNetworks)[index].Cidr))
+			ExpectWithOffset(1, cluster.MachineNetworks[index].ClusterID).To(Equal(*cluster.ID))
+			ExpectWithOffset(1, cluster.MachineNetworks[index].Cidr).To(Equal((*machineNetworks)[index].Cidr))
 		}
 		// TODO(MGMT-9751-remove-single-network)
-		Expect(cluster.MachineNetworkCidr).To(Equal(""))
+		ExpectWithOffset(1, cluster.MachineNetworkCidr).To(Equal(""))
```

Comment:
I don't understand the core of this offset appearing here. Could you please explain a bit?